Common/Schema/SchemaHelperND.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Exceptions;
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers;
+using Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Schema
+{
+    /// <summary>
+    /// Helper class to encapsulate interactions with the device schema.
+    ///
+    /// Elsewhere in the app we try to always deal with this flexible schema as dynamic,
+    /// but here we take a dependency on Json.Net where necessary to populate the objects
+    /// behind the schema.
+    /// </summary>
+    public static class SchemaHelperND
+    {
```

Comment:
this class has only 4 references. Can you actually change SchemaHelper to use generics instead of adding ND version ?